commons/addons/cellar-vs-fsbuckets.md

```diff
@@ -64,26 +64,26 @@ data you put in it.
 
 Cellar is an Amazon S3 API implementation using [Ceph Rados Gateway](https://ceph.com/ceph-storage/object-storage/).
 It provides an HTTP API that can be used with any S3 API Client.
-Clients exists for a lot of languages.
+Clients exist for a lot of languages.
 You can use any S3 Client, from the official Amazon ones to the community ones.
 
-If you already use S3, you will be in your element.
+If you already using S3, you will be in your element.
```

Comment:
Nope ^^
Read the sentence again ;)

Reply:
```suggestion
If you are already using S3, you will be in your element.
```

Reply:
I'll keep my version. It's still correct and it's shorter ;)